scipy/optimize/_qap.py

```diff
@@ -292,91 +281,71 @@ def _quadratic_assignment_faq(A, B,
-        and it must be doubly stochastic: each of its rows and columns must
-        sum to 1.
-
-        If unspecified or ``"barycenter"``, the non-informative "flat
-        doubly stochastic matrix" :math:`J = 1*1^T/m'`, where :math:`1` is
-        a :math:`m' \times 1` array of ones, is used. This is the "barycenter"
-        of the search space of doubly-stochastic matrices.
-
-        If ``"randomized"``, the algorithm will start from the
-        randomized initial search position :math:`P_0 = (J + K)/2`,
-        where :math:`J` is the "barycenter" and :math:`K` is a random
-        doubly stochastic matrix.
-
-    shuffle_input : bool (default = False)
-        To avoid artificially high or low matching due to inherent
-        sorting of input matrices, gives users the option
-        to shuffle the nodes. Results are then unshuffled so that the
-        returned results correspond with the node order of inputs.
-        Shuffling may cause the algorithm to be non-deterministic,
-        unless a random seed is set or an `rng` option is provided.
-
-    maxiter : int, positive (default = 30)
-        Integer specifying the max number of Franke-Wolfe iterations performed.
-
-    tol : float (default = 0.03)
-        A threshold for the stopping criterion. Franke-Wolfe
-        iteration terminates when the change in search position between
-        iterations is sufficiently small, that is, when the relative Frobenius
-        norm, :math:`\frac{||P_{i}-P_{i+1}||_F}{\sqrt{len(P_{i})}} \leq tol`,
+    partial_match : 2-D array of integers, optional (default: None)
+        Fixes part of the matching. Also known as a "seed" [2]_.
+
+        Each row of `partial_match` specifies a pair of matched nodes:
+        node ``partial_match[i, 0]`` of `A` is matched to node
+        ``partial_match[i, 1]`` of `B`. The array has shape ``(m, 2)``, where
+        ``m`` is not greater than the number of nodes, :math:`n`.
+
+    rng : int, `RandomState`, `Generator` or None, optional (default: None)
+        Accepts an integer as a seed for the random generator or a
+        ``RandomState`` or ``Generator`` object. If None (default), uses
+        global `numpy.random` random state.
+
+    P0 : "barycenter", "randomized", or a 2-D array (default: "barycenter")
+        Initial guess. Must be a doubly-stochastic matrix [3]_.
+
+        If ``"barycenter"`` (default), the initial position is the barycenter
+        of the Birkhoff polytope (the space of doubly stochastic matrices).
+        This is a :math:`m' x m'` matrix with all entries equal to
+        :math:1 / m'`.
+
+        If ``"randomized"`` the initial search position is
+        :math:`P_0 = (J + K) / 2`, where :math:`J` is the barycenter and
+        :math:`K` is a random doubly stochastic matrix.
+
+        If the initial position is an array, it must be a doubly stochastic
+        matrix of size :math:`m' x m'` where :math:`m' = n - m`.
```

Comment:
```suggestion
        matrix of size :math:`m' \times m'` where :math:`m' = n - m`.
```
Here and everywhere.

Reply:
Thanks. Fixed.